README.md

```diff
@@ -423,6 +422,82 @@ resp := &response{}
 err := c.RequestWithContext(ctx, "foo", req, resp)
 ```
 
+## JetStream Basic Usage
+
+```go
+import nats "github.com/nats-io/nats.go"
+
+// Connect to NATS
+nc, _ := nats.Connect(nats.DefaultURL)
```

Comment:
If its default URL is it needed? I can't remember..

Reply:
The function requires a string parameter, but we can pass an empty string for the default. Will update existing samples too.

Reply:
No need, that is good.